### ISSUE TITLE
Closing all windows in "Show all windows" should return user to the desktop.

### ISSUE THREAD

**Issue by jwoodard80** [linuxmint/cinnamon#7656]
```
 * Cinnamon version (cinnamon 3.8.4)
 * Distribution - (Mint 19)
 * Graphics hardware *and* driver used (NVIDIA Graphics)

**Issue**
Closing all windows using show all windows leaves a blank screen requiring "esc" key to go back to desktop


**Steps to reproduce**
Close all windows using "Show all windows" 


**Expected behaviour**
Unless I'm missing something, once I close all the windows there is nothing else that can be done from this screen. I'm expecting it to close and return me back to the desktop.


**Other information**

thank you for the consideration of this issue. :-)

**Comment by jwoodard80** [comment]
Couple of thoughts... One, my main concern was one of intuitiveness. Once I closed all the windows I was left with nothing and no indication of how to get back to the desktop.  Were I to be switching windows then it automatically goes to the selected window, no problem. However, with no windows I am left with nothing but a darkened screen. I assumed it was a bug until i started pressing keys and happened upon "Esc". I could see someone else less adept doing the same thing but semi-panicking as they are unsure what to do. :-)

Two, How would one know there is "switch workspaces" option as well? I would imagine the majority of users would be using this feature for a single workspace. There is an option to switch workspaces that is separate from this option. Granted I am one person, however, my instinctive thought would be that I would change workspaces then show all windows. Once all were closed on that view then it would take me back to the desktop of that workspace.

If we are leaning to leaving functionality as is for now, perhaps a message indicating functionality could be blended into the interface. "Press X to change workspaces or Esc to go to desktop" would at least provide some guidance for someone using the feature.

Lastly, I realize all suggestions are not accepted, however, I appreciate the consideration.

**Comment by collinss** [comment]
IMO at the very least scale should allow you to click on an empty spot to close it without selecting another window. I've been meaning to work on it (and expo too) and fix some of these little usability issues, but unfortunately I didn't get to it for this release. Maybe I'll get to it before the next one.

**Comment by jwoodard80** [comment]
Thank you for working to resolve this! Now that I know this is there and how to exit out I use it all the time, love it! Hopefully this change, at whatever point, will improve things overall. :-)

**Comment by flansuse** [comment]
Oh this would be a gem if it was implemented! The very first time I used Cinnamon, I thought this was unintuitive. It's expected to return you to the desktop when no more windows are open. I also expect to return to the desktop when I click on an empty space. Right now, the "Show all windows" feature still seems like an incomplete project, for the reasons mentioned in this report.

Secondly, part of its incompleteness lies in the fact that there is no highlight or focus when hovering over a window. The only indication is a tiny red X. KDE and GNOME Shell do this properly, and I think Cinnamon should mimic this tried-and-true behavior.

* Clicking an empty space should exit the "Show all windows" view.

* There should be an always visible "window" titled "Desktop", which does as expected: clicking it returns the user to the desktop.

*  Closing all windows should return the user to the desktop.

* Hovering over a window should highlight it or slightly bring it forward. (See KDE and GNOME Shell as an example of doing this properly)
